Edit Getting Started Section (#1133)
telerik/blazor-docs: getting-started/repl/snippets-and-scaffolders.md

# Telerik REPL for Blazor Snippets and Scaffolders

> To increase your productivity, Telerik REPL for Blazor provides code snippets for quickly adding and configuring a selection of UI components and scaffolders for configuring components in the REPL UI. 

Reviewer: Consider shorter and simpler sentences.

```suggestion
Telerik REPL for Blazor provides code snippets and scaffolders to increase your productivity. They enable you to quickly add and configure a selection of UI components in the REPL UI.
```

A complete list of the snippets for a specific component will be displayed in a popup once you start typing the component name in the editor.

REPL for Blazor allows you to add the snippets through the [**+** button](#add-button), by [dragging the selected component](#dragging-the-component), or [typing in the editor](#typing-in-the-editor).

### Add Button

> To add a component in the REPL editor, you can use the `+` button next to each snippet in the **Snippets and Scaffolders** menu. Clicking it adds the snippet at the current cursor position in the REPL editor. Then you can additionally configure the component and add the code block for the `C#` part.

Reviewer: ```suggestion
To add a component in the REPL editor, use the `+` button next to each snippet in the **Snippets and Scaffolders** menu. Clicking it adds the snippet at the current cursor position in the REPL editor. Then you can configure the component and add the code block for the `C#` part.
```

### Dragging the Component

> To directly insert a basic snippet with the desired component into the REPL editor, you can drag the component name from the **Snippets and Scaffolders** menu. The snippet will be inserted at the current cursor position. Then you can additionally configure the component and add the code block for the `C#` part.

Reviewer: ```suggestion
To directly insert a basic snippet with the desired component into the REPL editor, drag the component name from the **Snippets and Scaffolders** menu. The snippet will be inserted at the current cursor position. Then you can configure the component and add the code block for the `C#` part.
```
